index.html

```diff
@@ -12360,7 +12360,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
   &lt;li role="option" aria-setsize="16" <strong>aria-posinset="7"</strong>&gt; cantaloupes &lt;/li&gt;
   &lt;li role="option" aria-setsize="16" <strong>aria-posinset="8"</strong>&gt; dates &lt;/li&gt;
 &lt;/ul&gt;</pre>
-				<p>Authors MUST set the value for <pref>aria-posinset</pref> to an integer greater than or equal to 1, and less than or equal to the size of the set when that size is known. Authors SHOULD use <pref>aria-setsize</pref>.</p>
+				<p>Authors MUST set the value for <pref>aria-posinset</pref> to an integer greater than or equal to 1, and less than or equal to the size of the set when that size is known. If <pref>aria-posinset</pref> is specified, authors MUST specify a value for <pref>aria-setsize</pref>.</p>
```

Comment:
```suggestion
				<p>Authors MUST set the value for <pref>aria-posinset</pref> to an integer greater than or equal to 1, and less than or equal to the size of the set when that size is known. If <pref>aria-posinset</pref> is specified on an item, authors MUST specify a value for <pref>aria-setsize</pref> once on the container, or individually on all the items.</p>
```

Reply:
@carmacleod reminded us in the call that the spec excluded this on the container. IIRC, that was in the context of multi-level lists, since the some of the sub-list containers (and their levels) may not be rendered in the DOM.

Reply:
@cookiecrook so should I not accept the suggestion? Or is there more thinking to be done around the prose?

Reply:
@joanmarie @aleventhal @jcsteh what do you think about this proposed addition/change?

Reply:
What is the issue with specifying setsize on a container in a multi-level list? Obviously, you can only do this if a container with the appropriate scope exists. That is not true in all implementations, but that should not prevent it when such a container does exist.

Reply:
@mcking65 
> What is the issue with specifying setsize on a container in a multi-level list?

aria-setsize is not supported on containers. Whether or not it should be can be discussed in https://github.com/w3c/aria/issues/524, so that this PR is simply scoped to clarifying what is already there.